msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenByInteractiveFlowSupplier.java

```diff
@@ -108,6 +108,17 @@ private void updateRedirectUrl() {
     }
 
     private void openDefaultSystemBrowser(URL url) {
```

Comment:
nit cache the result to avoid all the string operations.

Reply:
Good point. Added an OSHelper class in the latest commit.